mongo/index_view_internal_test.go

```diff
@@ -215,32 +234,37 @@ func TestIndexView_CreateOneWithAllOptions(t *testing.T) {
 	_, coll := getIndexableCollection(t)
 	indexView := coll.Indexes()
 
+	indexOpts := options.Index().
+		SetBackground(false).
+		SetExpireAfterSeconds(10).
+		SetName("a").
+		SetSparse(false).
+		SetUnique(false).
+		SetVersion(1).
+		SetDefaultLanguage("english").
+		SetLanguageOverride("english").
+		SetTextVersion(1).
+		SetWeights(bsonx.Doc{}).
+		SetSphereVersion(1).
+		SetBits(2).
+		SetMax(10).
+		SetMin(1).
+		SetPartialFilterExpression(bsonx.Doc{}).
+		SetStorageEngine(bsonx.Doc{
+			{"wiredTiger", bsonx.Document(bsonx.Doc{
+				{"configString", bsonx.String("block_compressor=zlib")},
+			})},
+		})
+	// 4.9+ servers reject bucket size because they do not support geoHaystack indexes.
```

Comment:
Same question here. Does this branch run against `latest`/ `4.9`?

Reply:
See the response above.